_config/config.yml

```diff
@@ -1,12 +1,15 @@
 ---
 Name: subsiteconfig
 After:
- - 'framework/*'
- - 'cms/*'
+  - 'framework/*'
 ---
-AssetAdmin:
+SilverStripe\AssetAdmin\Controller\AssetAdmin:
   treats_subsite_0_as_global: true
 
 Director:
   rules:
-    'SubsiteXHRController': 'SubsiteXHRController'
+    SubsiteXHRController: SilverStripe\Subsites\Controller\SubsiteXHRController
```

Comment:
You could completely eliminate the custom rule by using the admin-specific route `admin/subsite_xhr`. CMS routing would provide the access point for you without having to register it manually.